### PR TITLE
More one home folder

### LINKED ISSUE #412: More one home folder
Hello

We can't write more one home folder :'(

### PR DESCRIPTION
**Pull request by MaximeMazet**
Hello,
I work is my own API and my own client with React and I've problem. I want to give in the url more one home folder but is not possible and I think i'm not the only one with this problem.

I've create the option for create in the configuration a special char and this special char is replace by / for create an other folder in the first if you want on readme bottom i've create an exemple.

Close #412 

Thanks a lot.

### CODE REVIEW

File: src/Controller/ElFinderController.php [hunk 1]
```diff
@@ -211,8 +213,14 @@ private function selectEditor(array $parameters, string $instance, string $homeF
...
     {
+        $efParameters = $this->container->getParameter('fm_elfinder');
+        //$multiHomeFolder = $efParameters['multi_home_folder'];
+        //$separator       = $efParameters['folder_separator'];
         $loader = $this->get('fm_elfinder.loader');
-        $loader->initBridge($instance); // builds up the Bridge object for the loader with the given instance
+        $loader->initBridge($instance, $efParameters); // builds up the Bridge object for the loader with the given instance
+
+
+
```
Comment (helios-ag) [inline src/Controller/ElFinderController.php:223]: Remove extra lines

File: src/Controller/ElFinderController.php [hunk 2]
```diff
@@ -211,8 +213,14 @@ private function selectEditor(array $parameters, string $instance, string $homeF
      */
     public function load(SessionInterface $session, EventDispatcherInterface $eventDispatcher, Request $request, string $instance, string $homeFolder): JsonResponse
     {
+        $efParameters = $this->container->getParameter('fm_elfinder');
+        //$multiHomeFolder = $efParameters['multi_home_folder'];
+        //$separator       = $efParameters['folder_separator'];
```
Comment (helios-ag) [inline src/Controller/ElFinderController.php:218]: Remove unused code

File: src/Controller/ElFinderController.php (hunk 1 above)
Comment (MaximeMazet) [reply src/Controller/ElFinderController.php]: I resolve your two suggestion ;)

File: src/Controller/ElFinderController.php (hunk 2 above)
Comment (MaximeMazet) [reply src/Controller/ElFinderController.php]: I remove this ;)

**Comment by MaximeMazet** [comment]
I'v done all change ;)

**Comment by helios-ag** [comment]
Hi @MaximeMazet, looks good, styleci isn't fully satisfied, but will make automatic PR from it.
Thanks

**Pull request merged**